scripts/circle/Dockerfile

```diff
@@ -1,36 +1,64 @@
-FROM centos:centos7.3.1611
-
-RUN yum -y update && \
-    yum -y install \
-        bzip2-devel.x86_64 \
-        curl \
-        git \
-        gcc \
-        gcc-c++ \
-        gcc-gfortran \
-        gmake \
-        hdf5-devel.x86_64 \
-        numpy.x86_64 \
-        python-devel.x86_64 \
-        https://github.com/openhpc/ohpc/releases/download/v1.3.GA/ohpc-release-1.3-1.el7.x86_64.rpm
-
-RUN yum -y update && \
-    yum -y install \
-        ohpc-base \
-        openmpi-gnu7-ohpc \
-        phdf5-gnu7-openmpi-ohpc \
-        netcdf-gnu7-openmpi-ohpc
+FROM centos:centos7
 
-WORKDIR /home/adios2
+# Install extra repos
+RUN yum -y install epel-release https://github.com/openhpc/ohpc/releases/download/v1.3.GA/ohpc-release-1.3-1.el7.x86_64.rpm && \
+    yum -y upgrade
```

Comment:
Is this what we want instead of `yum -y update`?  The dockerfile "best practices" documentation says to avoid upgrade, though they give the `apt-get` example rather than the `yum` one.

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run

Reply:
We can drop it entirely.  apt and yum work differently in this regard in that `apt-get update` is first required to update repo metadata before `apt-get install` will work.  `yum` on the other hand will automatically update it's metadata as needed.  The `yum upgrade` step is the equivalent of `apt-get dist-upgrade` and can be safely removed.

Reply:
Ok, I'll remove that.  Thanks.